core/signal.h

```diff
@@ -13,15 +13,21 @@
    limitations under the License.
 */
 /* Monitor for Remote Inspection. */
-#ifndef POSIX4WIN_H_
-#define POSIX4WIN_H_
-#ifdef WIN32
+#ifndef SIGNAL_H_
+#define SIGNAL_H_
 
+#include <signal.h>
```

Comment:
Should probably wrap the ```#include <signal.h>``` in a ```#ifndef WIN32``` to keep the original behavior in place.

Reply:
The place which include posix4win.h also include signal.h regardless the target OS/platform, so I think the code already assume that signal.h must exist on the target.
BTW, I found that CacheDebug reference posix4win.h directly here:
https://github.com/adamgreen/CrashDebug/blob/master/libCrashDebug/tests/mriPlatformBaseTest.h#L21
https://github.com/adamgreen/CrashDebug/blob/master/libCrashDebug/src/mriPlatform.c#L23
To avoid impact other project, look like I need keep posix4win.h as before, what do you think?

Reply:
Agreed. I will merge soon.

CrashDebug uses MRI as a submodule so it will be synced to an older commit/version. If I ever update it to a new version of MRI, I can just make the necessary changes to the CrashDebug codebase then. I suspect there have been other changes that I have made which will require changes to CrashDebug anyway.